test/functional/feature_elements_taproot_activation.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test Taproot soft fork activation for Elements
+
+Unlike Bitcoin, this (a) does not use Speedy Trial, and (b) does use
+a pair of new versionbits features which allows Taproot to have its
+own signalling period length (128 blocks on regtest) and activation
+threshold (100%).
+
+The primary purpose of this test is to confirm that this configuration
+works; the actual activation (e.g. how are Taproot-enabled transactions
+treated) is covered by other tests and by manual testing.
+
+There is an undocumented option `con_taproot_signal_start` which sets
+the block at which signalling starts; otherwise it is set to "always
+on" which means that signalling will not occur.
+"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class TaprootActivationTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def test_activation(self, node_idx, activation_height):
```

Comment:
In 0df8895b0ba53871fde3d42a766442b58439e389 "test: add test for Taproot activation"

It would be easier to read this test if instead of passing in the node index, the rpc object to use was passed in directly, e.g.:

```suggestion
    def test_activation(self, rpc, activation_height):
```

and the caller would be

```
        self.test_activation(self.nodes[0], 500)
```